doc/release/upcoming_changes/21468.new_feature.rst

```diff
@@ -0,0 +1,60 @@
+New function `np.show_runtime`
+------------------------------
+
+A new function ``np.show_runtime`` has been added to display the runtime
+information of the machine in addition to ``np.show_config`` which displays
+the build-related information.
+This function returns a list of dictionaries as follows::
```

Comment:
I think everything from here down should be removed. `show_runtime` should be added to the documentation here https://github.com/numpy/numpy/blob/58ee8a8bde14a84fa261ae329c3f99d953644f2c/doc/source/reference/routines.other.rst?plain=1#L47. Then the references above could change to `numpy.show_runtime`, and `numpy.show_config` (with single back-ticks), and they would be turned into links.

Reply:
Wow yeah, that's nice! Added via [706c922](https://github.com/numpy/numpy/pull/21468/commits/706c922ff278a6197f47ba678e8cc6128028d5ab)